vendors.json

```diff
@@ -118,12 +112,27 @@
     "constant": "GOCD",
     "env": "GO_PIPELINE_LABEL"
   },
+  {
+    "name": "Google Cloud Build",
+    "constant": "GOOGLE_CLOUD_BUILD",
+    "env": "BUILDER_OUTPUT"
+  },
   {
     "name": "LayerCI",
     "constant": "LAYERCI",
     "env": "LAYERCI",
     "pr": "LAYERCI_PULL_REQUEST"
   },
+  {
+    "name": "Gerrit",
+    "constant": "GERRIT",
+    "env": "GERRIT_PROJECT"
+  },
```

Comment:
Hello,

I am surprised to see Gerrit here. Does it have built-in CI support now? Cannot find anything mentioned at https://www.gerritcodereview.com/

Isn't `GERRIT_PROJECT` rather used by the Gerrit _Jenkins_ plugin(s)?